locale/es_ES/editor.xml

```diff
@@ -79,4 +79,5 @@
 	<message key="editor.internalReview.introduction">Durante la revisión interna, el editor/a asigna revisores/as internos a los archivos de envío y verifica los resultados de la revisión, antes de seleccionar la acción apropiada (la cual incluye notificar al autor/a): solicitar revisiones (revisiones hechas sólo por el editor/a), reenviar para revisión (se inicia otra ronda de revisiones), enviar para revisión externa (implica seleccionar los archivos para la revisión externa), aceptar el envío (implica seleccionar los archivos para la fase editorial), o rechazar el envío (archiva el envío).</message>
 	<message key="editor.externalReview.introduction">Durante la revisión externa, el editor/a asigna revisores/as externos a los archivos de envío y verifica los resultados de la revisión, antes de seleccionar la acción apropiada (la cual incluye notificar al autor/a): solicitar revisiones (revisiones hechas sólo por el editor), reenviar a revisión (se inicia otra ronda de revisiones), aceptar el envío (implica seleccionar los archivos para la fase editorial), o rechazar el envío (el editor/a archiva el envío).</message>
 	<message key="editor.submission.proof.manageProofFilesDescription">Cualquier fichero que ya haya sido subido en cualquier fase del envío puede añadirse al listado "Ficheros de prueba" marcando la siguiente casilla "Añadir" y pulsando "Buscar": todos los ficheros disponibles se listarán y podrán incluirse.</message>
+	<message key="editor.publicIdentificationExistsForTheSameType">El identificador público '{$publicIdentifier}' ya existe, asignado a otro objeto del mismo tipo. Por favor, elija identificadores únicos para los objetos del mismo tipo.</message>
```

Comment:
This need to be fixed.

"El identificador público '{$publicIdentifier}' ya existe y está asignado a otro objeto del mismo tipo. Por favor, elija identificadores únicos para los objetos del mismo tipo.